### PR TITLE
a workaround for an issue with SETTINGS[:libvirt] not being properly initialized in rpm environments

### PR DESCRIPTION
**Pull request by dmitri-d**

### CODE REVIEW

**Comment by ohadlevy** [comment]
looks good, can you please:
1. squash
2. add an upstream feature?
3. clean up commit messages (to include upstream ref) and whitespaces?

thanks

**Comment by ohadlevy** [comment]
@witlessbird ping?

**Comment by dmitri-d** [comment]
@ohadlevy pong! Will do all of the suggested things today. :)

**Comment by dmitri-d** [comment]
@ohadlevy ping?

**Comment by domcleal** [comment]
[test] again, Jenkins got stuck.

File: config/boot.rb [hunk 1]
```diff
@@ -3,21 +3,7 @@
...
-      SETTINGS[:libvirt] = true
-    else
-      SETTINGS[:libvirt] = false
-    end
-  rescue LoadError
-    print "Libvirt bindings are missing - hypervisor management is disabled"
-    SETTINGS[:libvirt] = false
-  end
-end
+# setting of SETTINGS[:libvirt] is delayed until config/application.rb,
```
Comment (domcleal) [inline config/boot.rb]: I'd just remove this, it's more PR or commit message detail.

**Comment by domcleal** [comment]
@witlessbird could you ensure the commit message fits the normal style?

```
fixes #1234 - message
```

The format's important as redmine picks it up and links it to the ticket.  Cheers!

**Comment by dmitri-d** [comment]
@domcleal: done and done

**Comment by domcleal** [comment]
Merged as 32bf6e0.  Thanks @witlessbird!

**Pull request closed without merging**